R/query_dataset.R

```diff
@@ -130,28 +140,30 @@
 #'   example_id(group = "attendance"),
```

Comment:
This line just above looks stray and like it shouldn't be there?

#' example_id("filter_items_short", group = "attendance")

Reply:
That's just to demo what the input looks like, I've tried to make it more obvious now.